application/core/Traits/LSApplicationTrait.php

```diff
@@ -31,7 +30,7 @@ public function getCurrentUserId()
         }
         /* use App()->session and not App()->user fot easiest unit test */
         $this->currentUserId = App()->session['loginID'];
-        if ($this->currentUserId && !User::model()->findByPk($this->currentUserId)) {
+        if ($this->currentUserId && !User::model()->active()->findByPk($this->currentUserId)) {
```

Comment:
Cool

Reply:
`$this->currentUserId && ` --> Wouldn't be good to move this to an independent if?
Something like 
```
if (!$this->currentUserId || empty($this->currentUserId)) {
  return 0
}
```

That would make code a bit less complex

Reply:
Unsure : more clear ?

```
if (empty($this->currentUserId)) {
  return $this->currentUserId;
}
```

before ?

Reply:
MASTER only ;)

Reply:
I think would be better, yes

Reply:
Sorry didn't see your answer before merge …